### ISSUE TITLE
More granular notification settings.

### ISSUE THREAD

**Issue by alloy** [librariesio/support#89]
Currently I’m monitoring 2 repos and watching 1 library. I have disabled email notifications, because I don’t want emails for all the notifications that come from the repo monitoring (I use the web hooks for those), but I _do_ want an email notification for the library I’m watching.

The level of granularity doesn’t need to be too advanced for me. I think different settings for ‘repository monitoring’ and ‘subscriptions’ would be enough, because they are very different in purpose and in number of notifications you would receive.

**Comment by andrew** [comment]
@alloy agreed, that would be useful, need a way to manage notification settings per monitored repo

**Comment by ldodds** [comment]
I'm disabling emails for the repo I'm watching as there are too many emails. For my purposes I'd prefer a weekly or monthly digest. I should probably be checking more frequently, but I'm more likely to action that apps that I'm not working on daily.
